scripts/linters/pylint_extensions_test.py

```diff
@@ -3340,3 +3340,183 @@ def __init__(self, fake_arg):
 
         with checker_test_object.assertNoMessages():
             checker_test_object.checker.visit_classdef(metaclass_node)
+
+
+class DisallowHandlerWithoutSchemaTests(unittest.TestCase):
```

Comment:
Is this a stacked PR?

Reply:
@sajalasati this pr is stacked with the changes from another PR. This is because the original pr is delayed and is yet to be merged. So this PR is created so that I can take an initial pass. Once the original PR is merged, the changes will be made here. Sorry for the confusion.